index.js

```diff
@@ -19,6 +19,7 @@ import {list} from './list.js'
 
 // Rules.
 const source = 'retext-contractions'
+const url = 'https://github.com/retextjs/retext-contractions#readme'
```

Comment:
Is this something we could read or construct from the package.json file?

Reply:
Yes, but AFAIK that would add a dependency on NodeJS.

Reply:
I thought that import assertions were supported in browser/deno
https://v8.dev/features/import-assertions
https://deno.com/blog/v1.17#import-assertions-and-json-modules

They are new, so maybe we hold off a bit, but there is a cross platform solution now.

Reply:
That would be a great solution, but not for now. Not even NodeJS 16 supports it.

The same could be said for `source`, which matches `name` in `package.json`.

Also the value isn’t likely to change, so I don’t think this is a big issue.